#!/usr/bin/env python

PACKAGE = "roseli"

import roslib
roslib.load_manifest(PACKAGE)
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_value_ocr", int_t, 0, "Minimum value of the HSV image of tag (Value)", 0, 0, 255)
gen.add("min_saturation_ocr", int_t, 0, "Minimum value of the HSV image of tag (Saturation)", 140, 0, 255)
gen.add("min_hue_ocr", int_t, 0, "Minimum value of the HSV image of tag (Hue)", 100, 0, 255)
gen.add("max_value_ocr", int_t, 0, "Maximum value of the HSV image of tag (Value)", 255, 0, 255)
gen.add("max_saturation_ocr", int_t, 0, "Maximum value of the HSV image of tag (Saturation)", 255, 0, 255)
gen.add("max_hue_ocr", int_t, 0, "Maximum value of the HSV image of tag (Hue)", 150, 0, 255)

exit(gen.generate(PACKAGE, "readtag", "ocr_tag"))
